compiler/src/iree/compiler/Codegen/LLVMCPU/test/pipeline_tests.mlir

```diff
@@ -358,23 +358,23 @@ hal.executable private @quant_matmul_fusion {
         %c-128_i32 = arith.constant -128 : i32
         %c127_i32 = arith.constant 127 : i32
         %c0_i32 = arith.constant 0 : i32
-        %0 = hal.interface.binding.subspan set(0) binding(0) type(storage_buffer) alignment(64) offset(%c0) : !flow.dispatch.tensor<readonly:tensor<2304x24xi8>>
-        %1 = hal.interface.binding.subspan set(0) binding(1) type(storage_buffer) alignment(64) offset(%c0) : !flow.dispatch.tensor<readonly:tensor<24x144xi8>>
```

Comment:
Why did you have to change the test input? I can see why the test output is updated in [select_x86_64_lowering_strategy.mlir](https://github.com/openxla/iree/pull/16041/files#diff-1f921a289d33fa2f864c99c0664b1084e10785199c74897513df61e9bec77060), but not sure about this change.

Reply:
The size is too small so the trivial loops (i.e., one trip loops) are folded away. The test was added a long time ago, I think this is what it wants to test.